pilot/pkg/proxy/envoy/v2/eds_sh_test.go

```diff
@@ -66,36 +70,36 @@ func TestSplitHorizonEds(t *testing.T) {
 	initRegistry(4, []string{}, 4)
 
 	// Update cache
-	pilotServer.EnvoyXdsServer.ClearCache()
+	server.EnvoyXdsServer.ClearCache()
 
 	// Verify that EDS from network1 will return 1 local endpoint with local VIP + 2 remote
 	// endpoints weighted accordingly with the IP of the ingress gateway.
 	verifySplitHorizonResponse(t, "network1", sidecarId("10.1.0.1", "app3"), expectedResults{
 		numOfNetworks:  3,
 		localEndpoints: []string{"10.1.0.1"},
-		remoteWeights:  map[string]uint32{"159.122.219.2": 2, "159.122.219.3": 3},
+		remoteWeights:  map[string]uint32{"159.122.219.2": 43, "159.122.219.3": 64},
```

Comment:
These are not the right values for remote endpoints.
Their weights are supposed to be the number of endpoints for the EDS cluster.

Reply:
There is a normalization. Because the valid range is [1, 128]

Reply:
Not sure I understand what you mean because 2 and 3 for instance are in the [1,128] range.

Remote endpoints are implemented based on https://docs.google.com/document/d/1E3cTCJiCrVjKdjI7pX5rU0MAw99xCq8UyfZKCu8LDYw/edit#heading=h.qex63c29z2to
so the desired weights are according to the description there.